lib/torus_direct.dart

```diff
@@ -18,26 +19,72 @@ enum TorusLogin {
   jwt
 }
 
+enum AggregateVerifierType { single_id_verifier }
```

Comment:
I think we should use camel case, most Flutter/Dark official examples use camel case values in enum. This makes a better DX.

Reply:
I intends to match the pattern used in other sdks here, also does not really seems to support assigning a string value to a enum.